### PR TITLE
wdk-csswizardry-grids

### PR DESCRIPTION
**Pull request by benjaminparry**
As per https://github.com/createdotnet/dyos_html/pull/699

### CODE REVIEW

**Comment by benjaminparry** [comment]
@AidanThreadgold @createdotmatt ~ Grid docs

**Comment by mattgreenfield** [comment]
Looks good

**Comment by mattgreenfield** [comment]
Change log?

**Comment by benjaminparry** [comment]
@createdotmatt ~ Changelog done

File: _tutorials/requirements-restrictions.md [hunk 1]
```diff
@@ -41,15 +41,69 @@ There are a few tags that are required in order for your template to be processe
 These are listed below.
 
 
-#### Core Styling
-This tag should be placed in the head of your document in order to import the default CSS stylings. Your custom CSS stylesheet(s) should be placed below this tag.
+### Core Styling
+
+To style the core components, patterns and widgets in the Create.net system we provide a vanilla (un-themed) version of our CSS boilerplate. This tag should be placed in the head of your document in order to import the default CSS styling. Your custom CSS stylesheet(s) should be placed below this tag.
```
Comment (AidanThreadgold) [inline _tutorials/requirements-restrictions.md]: Create.net system or Create.net templates?

**Comment by AidanThreadgold** [comment]
@benjaminparry Okay to merge today?

**Comment by benjaminparry** [comment]
Yeah I think so.

**Pull request merged**